Openshift/documentation/scripts/docDataExtracter.sh

```diff
@@ -90,6 +92,28 @@ oc get identities
 oc get pv > pv.log
 oc get sc > sc.log
 
+###### Gluster related info 
+
+oc get storageclass > storageclass.log
+
+#gluster storage data
+oc get pvc -n default > pvc.log
+
+#Verify registry using gluster
+oc describe dc/docker-registry -n default | grep -A3 Volumes > registryVerification.log
+
```

Comment:
adding a step to install heketi-cli? Or should we add a comment that scripts assumes cli being installed on the bastion?

Reply:
Will add comment.